thinkhazard/static/less/report.less

```diff
@@ -61,3 +77,49 @@ body {
     }
   }
 }
+
+#location {
+  .legendHeading(18px);
+  margin: 0px 20px 27px;
+}
+
+#level {
+  .legendHeading(24px);
+  margin: 4px 20px 0px;
+}
+
+.legendHeading(@font-size){
+  color: @text-color;
+  font-size: @font-size;
+  font-family: @font-semibold;
+}
+
+#mapBlock{
+  @bg-color: #FFF;
+  background-color: @bg-color;
+  padding: 10px;
+  .box-shadow(10px 10px 10px 0px rgba(0, 0, 0, 0.33));
+}
+
+#dataSource{
+  padding: 27px 0px 30px;
+  margin: 0 20px;
+  border-top-style: solid;
+  border-top-width: 1px;
+  border-top-color: #e6e6e6;
+}
+
+#dataSource h5{
+  font-size: 14px;
+  color: @text-color;
+  text-transform: uppercase;
+  margin-bottom: 4px;
+  font-family: @font-semibold
+}
+ 
+#dataSource div{
+  font-size: 14px;
+  color: @text-color;
+  font-family: @font-regular;
+}
```

Comment:
The `h5` and `div` blocks could go in the `#dataSource` block, but I'll let @pgiraud comment about this.